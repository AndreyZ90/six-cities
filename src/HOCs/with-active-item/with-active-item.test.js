import React from 'react';
import TestRenderer from 'react-test-renderer';

import withActiveItem from '@/HOCs/with-active-item/with-active-item';

const MockComponent = () => <div className="wrapped-component">Wrapped Component</div>;
const WrappedComponent = withActiveItem(MockComponent);

describe(`withActiveItem HOC snapshot`, () => {
  test(`Should correctly render withActiveItem HOC`, () => {
    const tree = TestRenderer.create(<WrappedComponent />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
