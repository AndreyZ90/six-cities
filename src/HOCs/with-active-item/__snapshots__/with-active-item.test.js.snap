// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withActiveItem HOC snapshot Should correctly render withActiveItem HOC 1`] = `
<div
  className="wrapped-component"
>
  Wrapped Component
</div>
`;
